src/core/propertyTypes.js

```diff
@@ -117,6 +118,10 @@ function srcParse (value) {
   var el = selectorParse(value);
   if (el) { return el.getAttribute('src'); }
 
+  if (value.charAt(0) !== '#') {
+    warn('Value provided for `src` attribute is not valid: ' + value + '. Value must be selector notation or wrapped in `url()`.');
```

Comment:
Should we mention this is either ID or `url()`, rather than just "selector notation"?